products/telegraf.md

```diff
@@ -245,7 +245,10 @@ releases:
 > [Telegraf](https://github.com/influxdata/telegraf) is an agent for collecting, processing,
 > aggregating, and writing metrics.
 
-Telegraf is based on a plugin system, and this page only covers the Telegraf server.
-Only the latest two minor releases are supported with maintenance patch releases.
-Compatibility is not guaranteed across major releases. Each major release is supported for a
-minimum of 12 months.
+InfluxData have not published specific support periods for their OSS products, although their
```

Comment:
I can see an OSS version for Influxdb, but not for Telegraph. Is there really disinct Telegraph OSS / commercial versions ?